hpat/utils.py

```diff
@@ -531,3 +532,7 @@ def dump_node_list(node_list):
 
 def debug_prints():
     return numba.config.DEBUG_ARRAY_OPT == 1
+
+def update_globals(func, glbls):
```

Comment:
it looks like this function used nowhere. please delete

Reply:
It is used in distributed, dataframe, high frame typad & untyped passes. It updates globals in generated functions. It was introduced in order to restore functionality of ```inline_closure_call``` which doesn't update globals anymore.

You can see details in related commit: https://github.com/IntelPython/hpat/pull/224/commits/4b828c921ee19b3b12667f573eaab4105a91e22a